packages/vscode-extension/src/extension.ts

```diff
@@ -1011,8 +1010,6 @@ async function runBackgroundAsyncTasks(
   await handlers.postUpgrade();
   const upgrade = new ExtensionUpgrade(context);
   upgrade.showChangeLog();
-  const preview = new PrereleasePage(context);
```

Comment:
do we need to remove all pre-release related code? Or just remove checkAndShow? Since preview logic maybe still usable later. We just need refine it right?

Reply:
sure